deploy.yml

```diff
@@ -51,80 +54,81 @@ stages:
                 downloadPath: '$(Build.StagingDirectory)/packages'
               displayName: 'Retrieve built conda package'
             - bash: |
-                anaconda --token "$ANACONDA_TOKEN" upload --user scipp --label ${{ parameters.conda_label }} '$(Build.StagingDirectory)/packages/${{ package.Key }}/'scipp*.tar.bz2
+                anaconda --token "$ANACONDA_TOKEN" upload --user scipp --label ${{ parameters.conda_label }} '$(Build.StagingDirectory)/packages/${{ package.Key }}/'scipp*.tar.bz2 --force
               env:
                 ANACONDA_TOKEN: $(anaconda_token_secret)
-              displayName: 'Deploy ${{ package.Key }}'
-
-          - task: DownloadBuildArtifacts@0
-            inputs:
-              buildType: 'current'
-              specificBuildWithTriggering: true
-              downloadType: 'single'
-              artifactName: 'documentation'
-              downloadPath: '$(Build.StagingDirectory)'
-            displayName: 'Retrieve built documentation'
-          - task: InstallSSHKey@0
-            inputs:
-              knownHostsEntry: 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ=='
-              sshPublicKey: '$(gh_pages_deploy_key_public)'
-              sshKeySecureFile: 'gh_pages_deploy_key'
-            displayName: 'Add GitHub Pages deploy key'
-          - bash: |
-              set -ex
-              # Publishing documentation involves a Git commit, this sets the identity for that commit
-              git config --global user.name 'Documentation Builder'
-              git config --global user.email 'nobody@localhost'
-            displayName: 'Git config'
-          - ${{ if eq(length(parameters.gh_pages_subdirectory), 0) }}:
+              displayName: 'Deploy to anaconda cloud ${{ package.Key }}'
+          - ${{ if eq(parameters.publish_docs, true) }}:
+            - task: DownloadBuildArtifacts@0
+              inputs:
+                buildType: 'current'
+                specificBuildWithTriggering: true
+                downloadType: 'single'
+                artifactName: 'documentation'
+                downloadPath: '$(Build.StagingDirectory)'
+              displayName: 'Retrieve built documentation'
+          - ${{ if eq(parameters.publish_docs, true) }}:
```

Comment:
Why do you need a second if statement here? can't you put everything inside the same `if` block?

Reply:
Not without significant rewrite I think? First task (anaconda upload) we still want. second two tasks may be disabled.